doc/using/organizing.chapter.md

```diff
@@ -0,0 +1,96 @@
+# Organizing packages with callPackage {#chap-organizing}
```

Comment:
In the documentation team we've established that the Nixpkgs manual should only serve as reference documentation. This however is not really reference docs, but rather a [Tutorial or a Guide](https://nix.dev/contributing/documentation/diataxis), which should go to https://nix.dev ([source](https://github.com/NixOS/nix.dev)) instead. Could you PR this file to there instead?

Reply:
Could you update the `CONTRIBUTING.md` or `doc/README.md` to update this guidance? It would've saved me time writing and formatting this here, and it'll scale better than asking each relevant PR to move.

If tutorials and guides live in nix.dev and not in the manuals, https://nixos.org/learn should say that and we should make an [announcement post](https://nixos.org/blog/announcements) to explain the change. The last time I checked, nix.dev was [the "Nix community cookbook"](https://github.com/NixOS/nix.dev/blob/7c4daba3ad01f2646f02f60a652fed9ca4508573/source/index.rst), not the source for all Nix documentation.

Reply:
> Could you update the `CONTRIBUTING.md` or `doc/README.md` to update this guidance?

Good point, yes: https://github.com/NixOS/nixpkgs/pull/272183

No worries about the formatting though, nix.dev also uses Markdown with only minor differences.

> If tutorials and guides live in nix.dev and not in the manuals, https://nixos.org/learn should say that and we should make an [announcement post](https://nixos.org/blog/announcements) to explain the change. The last time I checked, nix.dev was [the "Nix community cookbook"](https://github.com/NixOS/nix.dev/blob/7c4daba3ad01f2646f02f60a652fed9ca4508573/source/index.rst), not the source for all Nix documentation.

Yeah things are changing, nix.dev became the official home for documentation not too long ago. Slowly though, things still need to be figured out, but that's what the docs team is working towards.

Reply:
There's a bit of reference level docs in here that really need to stay, since none of this stuff is currently documented at all. I have a WIP post that mentions makeScope which I had to read sources to write because there's no docs.

I'm going to reach out to you on Matrix about stuff we should do to improve the explanation of where things belong, because the current state of confusion is untenable.